src/ray/stats/stats.h

```diff
@@ -74,6 +74,7 @@ static inline void Init(const TagsType &global_tags, const int metrics_agent_por
     RAY_LOG(INFO) << "Disabled stats.";
     return;
   }
+  RAY_LOG(DEBUG) << "Initialize stats";
```

Comment:
`Initializing stats.`
Maybe also add `Stats initialized.` at the end of this function.